### ISSUE TITLE
How to replace favicon.ico icon？

### ISSUE THREAD

**Issue by aaaaaa-xiang** [lobehub/sd-webui-lobe-theme#567]
### 🥰 需求描述 | Feature Description

![screenshot-1715828663602](https://github.com/lobehub/sd-webui-lobe-theme/assets/84775435/ac411606-07ed-490c-adf4-f40dfad9ecbd)
![screenshot-1715828699919](https://github.com/lobehub/sd-webui-lobe-theme/assets/84775435/cfeee313-b0ab-46c5-a108-8eee974acd48)
I have obsessive-compulsive disorder and like to upload my favorite icons。

### 🧐 解决方案 | Proposed Solution

Please tell me how to operate it. I really need it.

**Comment by YangWudi233** [comment]
me too

**Comment by crsongbird** [comment]
Yeah, I'm with you guys. I like the theme (mostly, but it needs some medium-dark colors between jet black and neutral grey) and I loathe this icon and branding.
